### PR TITLE
13 DSK cards (Diversion Specialist and friends)

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/diversion_specialist.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Diversion Specialist
+ManaCost:3 R
+Types:Creature Human Warrior
+PT:4/3
+K:Menace
+A:AB$ Dig | Cost$ 1 Sac<1/Creature.Other;Enchantment.Other/another creature or enchantment>
+ | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top card of your library. You may play that card this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/diversion_specialist.txt:7]: bad break

File: forge-gui/res/cardsfolder/upcoming/sawblade_skinripper.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Types:Creature Human Assassin
+PT:3/2
+K:Menace
+A:AB$ PutCounter | Cost$ 2 Sac<1/Creature.Other;Enchantment.Other/another creature or enchantment> | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on CARDNAME.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | SVarCompare$ GT0 | Execute$ TrigDealDamage | TriggerDescription$ At the beginning of your end step, if you sacrificed one or more permanents this turn, CARDNAME deals that much damage to any target.
+SVar:TrigDealDamage:DB$ DealDamage | NumDmg$ X | ValidTgts$ Any
+SVar:X:PlayerCountPropertyYou$SacrificedThisTurn Permanent
+DeckHints:Ability$Investigate
+DeckHas:Ability$Investigate|Token
+Oracle:Menace\n{2}, Sacrifice another creature or enchantment: Put a +1/+1 counter on Sawblade Skinripper\nAt the beginning of your end step, if you sacrificed one or more permanents this turn, Sawblade Skinripper deals that much damage to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sawblade_skinripper.txt:12]: .\n

File: forge-gui/res/cardsfolder/upcoming/splitskin_doll.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Splitskin Doll
+ManaCost:1 W
+Types:Artifact Creature Toy
+PT:2/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters, draw a card. Then discard a card unless you control another creature with power 2 or less.
+SVar:TrigDraw:DB$ Draw | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | NumCards$ 1 | Mode$ TgtChoose | IsPresent$ Creature.Other+YouCtrl+powerLE2 | PresentCompare$ EQ0
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/splitskin_doll.txt:7]: bad condition

File: forge-gui/res/cardsfolder/upcoming/twist_reality.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:Twist Reality
+ManaCost:1 U U
+Types:Instant
+A:SP$ Charm | Choices$ DBCounter,DBDread | Defined$ You
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/twist_reality.txt:4]: Defined again

File: forge-gui/res/cardsfolder/upcoming/splitskin_doll.txt (hunk 3 above)
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/splitskin_doll.txt]: Honestly, is there a reason for having so many slightly different syntaxes?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/splitskin_doll.txt]: you could search if there's a card that first has a restriction (IsPresent etc.) to cast/activate and then a resolving condition on the same line (ConditionCheckSVar, ...)
that would be an answer

**Review by tool4ever — APPROVED** [review]

**Pull request merged**